tests/test_modules/data/website_load/js/package.json

```diff
@@ -25,8 +25,8 @@
   },
   "homepage": "https://github.com/AutolabJS/AutolabJS/wiki",
   "dependencies": {
-    "jquery": "^3.3.1",
-    "file-saver": "^1.3.2",
+    "jquery": "3.3.1",
```

Comment:
Is there any reason for changing to precise versions here?

Reply:
Filesaver has updated to 1.3.4 and that caused builds to fail today. To avoid that, the precise file versions have been used. The package-lock.json files were also included for the same reason.